wave_optimization/CMakeLists.txt

```diff
@@ -5,38 +5,50 @@ IF (CMAKE_COMPILER_IS_GNUCXX)
   ADD_DEFINITIONS(-std=gnu++0x)
 ENDIF()
 
+# PACKAGES
+FIND_PACKAGE(Ceres REQUIRED)
+
 # INCLUDES
```

Comment:
Actually second thought, can we delay this for another PR? It's proving to be quite complicated